MoltenVK/MoltenVK/Commands/MVKCommandEncoderState.mm

```diff
@@ -322,6 +322,11 @@
 	getContent(_isPolygonModePoint, isDynamic) = (polygonMode == VK_POLYGON_MODE_POINT);
 }
 
+void MVKRenderingCommandEncoderState::setLineWidth(float lineWidth, bool isDynamic) {
+	auto mtlLineWidth = lineWidth;
+	setMTLContent(LineWidth);
```

Comment:
unused local variable `mtlLineWidth`?

Reply:
It's part of the setup for the `setMTLContent(LineWidth)` macro pattern. See other nearby uses.

Reply:
Just checked again and saw it was a macro specific to the file. I mixed it up with the `setContent` functions from the command buffer classes.